cla_public/templates/checker/result/confirmation.html

```diff
@@ -41,9 +43,10 @@
 
   {% if session.stored.callback_requested %}
     <p>
-      {% trans %}Your details have been submitted and an operator will try to call
-        you back around the time you requested {% endtrans %}
-        (<strong>{{ session.stored.callback_time|datetime('full') }}</strong>).
+      {{ _('Your details have been submitted and an operator will try to call') }}
```

Comment:
probably not a great idea to split a sentence into separate translated strings – the sentence order may not work correctly in welsh and the lack of context might lead to incorrect translation of the fragments. (applies to a few places in this file; but template translations need a review throughout the site anyway)